crates/pgt_lsp/src/capabilities.rs

```diff
@@ -67,3 +61,13 @@ pub(crate) fn server_capabilities(capabilities: &ClientCapabilities) -> ServerCa
         ..Default::default()
     }
 }
+
+/// Returns all available command IDs for capability registration.
+/// Since CommandActionCategory has variants with data, we can't use strum::IntoEnumIterator.
+/// Instead, we manually list the command IDs we want to register.
+fn available_command_ids() -> Vec<String> {
+    vec![
+        "postgres-tools.executeStatement".to_string(),
```

Comment:
The ID in `pgt_lsp/src/handlers/code_actions.rs` also needs to be adjusted 👍

Reply:
Also, we're now just sending the full statement over to the client anyway, so we could just as well change the CommandActionCategory::ExecuteStatement(_) to hold a `String` instead of a `StatementId` – I think you can then use strum again :)

Reply:
ah good point!

Reply:
reverted my changes. I think its simpler to just add a comment to the default implementation